backend-api/infra/parent.yaml

```
@@ -49,12 +49,10 @@ Parameters:
     Default: false
 
   LambdaCanaryDeployment:
-    Description: Specifies the configuration to enable gradual Lambda deployments
+    Description: |
+      Specifies the configuration to enable gradual Lambda deployments. Value can be 'AllAtOnce', a default deployment config, or name of custom DeploymentConfig.
     Type: String
-    AllowedValues:
-      - AllAtOnce
-      - Linear20PercentEvery1Minute
-    Default: AllAtOnce
+    Default: none
```

```suggestion
    Default: None
```

These are usually case sensitive. https://github.com/govuk-one-login/devplatform-deploy/tree/main/sam-deploy-pipeline#:~:text=String-,None,-Canary%20strategy%20to%20be%20used%20with%20your%20Lambda

---

ooooof good spot